app/actions/contactsActions.js

```diff
@@ -5,12 +5,73 @@ import { has, isEmpty, keys, values, indexOf, zipObject, omit } from 'lodash-es'
 
 import { getStorage, setStorage } from '../core/storage'
 
+export const ID = 'contacts'
+
+export type TChains = { contactKey: string, chain: string }[]
+
+const STORAGE_KEY_CHAIN = 'chainBook'
+const STORAGE_KEY = 'addressBook'
+
+export const getChains = async (): Promise<TChains> =>
+  getStorage(STORAGE_KEY_CHAIN)
+
+const setChainContact = async (contactKey: string, chain: string) => {
```

Comment:
If we already have chain as a param why do we call `getChains` on the next line?